contrib/docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres:11
+    environment:
+      - POSTGRES_PASSWORD=pg_pwd!
+      - POSTGRES_DB=test_db
+    ports:
+     - "5432"
+
+  mssql:
+    build:
+      context: .
+      dockerfile: Dockerfile-mssql
+    ports:
+      - "1433"
+
+  mysql:
+      image: mysql:latest
+      environment:
+        - MYSQL_ROOT_PASSWORD=MysqlPassword
+        - MYSQL_DATABASE=test_db
+      ports:
+        - "3306"
+
+  turbodbc:
+    build:
+      context: .
+      dockerfile: Dockerfile-turbodbc
+    environment:
+       - ODBCSYSINI=/turbodbc/contrib/docker
```

Comment:
Put these environment specifications in the docker file (at the end)

Reply:
Makes sense for LC_ALL, VIRTUAL_ENV and PATH, but IMO ODBCSYSINI, PYTHONPATH belong to `docker-compose.yml`, because only here the bind-mount location `/turbodbc` is defined these variables refer to.